Documentation/bpf/architecture.rst

```diff
@@ -81,7 +81,7 @@ The advantages for pushing these instructions into the kernel include:
   programs, for example, reuse the existing in-kernel drivers and operate on the
   provided DMA buffers containing the packet frames without exposing them or an entire
   driver to user space as in other models. Moreover, XDP programs reuse the existing
-  stack instead of bypassing it. BPF can be considered a generic "glue code" to
+  stack instead of bypassing it. BPF can be considered a generic "glue code" for
```

Comment:
Double `for` makes it hard to parse, maybe?

```suggestion
  stack instead of bypassing it. BPF can be considered a generic "glue code" between
```